README.md

```diff
@@ -8,7 +8,7 @@ Okta Spring Boot Starter
 
 Okta's Spring Boot Starter will enable your Spring Boot application to work with Okta via OAuth 2.0/OIDC.
 
-**NOTE:** This library works with Spring Boot 2.2+. If you need support for Spring Boot 1.5.x, use version 0.6.
+**NOTE:** This library works with Spring Boot 2.4+. If you need support for Spring Boot 1.5.x, use version 0.6.
```

Comment:
@bdemers should we add another line that says `if you need support for Spring Boot 2.2.x, use version 1.4.0`?

Reply:
Yeah, maybe we should break it out into a grid, maybe something similar to our other READMEs "release" section:
https://github.com/okta/okta-sdk-java#release-status

Reply:
updated.